piazza_api/network.py

```diff
@@ -358,6 +358,54 @@ def delete_post(self, post):
 
         return self._rpc.content_delete(params)
 
+    def add_feedback(self, post):
+        """Marks a post as a good note
+        :type post: dict|str|int
+        :param post: Either the post dict returned by another API method, the post ID, or
+            the `cid` field of that post.
+        :rtype: dict
+        :returns: Status `'OK'` if adding feedback was successful.
+         """
+
+        try:
+         cid = post['id']
+        except KeyError:
+         cid = post
+        except TypeError:
+            post = self.get_post(post)
+            cid = post['id']
+
+        params = {
+            "cid": cid,
+            "type": "tag_good"
+        }
+
+        return self._rpc.content_add_feedback(params)
+
+    def remove_feedback(self, post):
+        """Marks a post as a good note
```

Comment:
Unmarks*

Reply:
> 
> 
> Thanks for the contribution! Just a couple nits in the docstrings to fix then this should be good to go.

Ah, thanks for catching those. Definitely was typing/copy-pasting those docstrings too quickly. Also, a huge thanks for developing this API. It's been very handy as a TA and easy to use!